src/apint/constructors.rs

```diff
@@ -189,6 +190,18 @@ impl ApInt {
         }
     }
 
+    pub fn from_vec_u64(val: Vec<u64>) -> Option<ApInt> {
```

Comment:
I really do not want to have this as public function to be honest. At least we should be very cafeful about this public API addition. Maybe we could stabilize it later for public access. It is too special, reflects too much of the inner working to the outside and the `Option<ApInt>` return type should also be more in par with the rest of the library and return a `Result` instead.

Reply:
Also needs proper documentation.

Reply:
You're right. I will make it private for now